Resources/config/format_listener.xml

```diff
@@ -22,7 +22,7 @@
             <argument type="collection" /> <!-- mime types -->
         </service>
 
-        <service id="fos_rest.request_matcher" class="Symfony\Component\HttpFoundation\RequestMatcher" public="false" />
+        <service id="fos_rest.format_request_matcher" class="Symfony\Component\HttpFoundation\RequestMatcher" public="false" />
```

Comment:
this is just a cosmetic change as we now have 2 different places where we use a RequestMatcher